src/test/java/com/conveyal/datatools/manager/controllers/api/FeedSourceControllerTest.java

```diff
@@ -119,6 +150,14 @@ private static FeedSource createFeedSource(String name, URL url) {
         return feedSource;
     }
 
+    /**
+     * Helper method to create label
+     */
+    private static Label createLabel(String name) {
+        Label label = new Label(name, "A label used during testing", "#123", false, project.id);
```

Comment:
Can be inlined. e.g.:

`        return new Label(name, "A label used during testing", "#123", false, project.id);
`

Reply:
Who knows how this happened thanks!